packages/zowe-explorer/__tests__/__unit__/shared/init.unit.test.ts

```diff
@@ -145,58 +143,26 @@ describe("Test src/shared/extension", () => {
                 parm: [{ isDirty: false, fileName: "_", uri: vscode.Uri.parse("") }],
                 mock: [],
             },
-            {
-                name: "onDidSaveTextDocument:2",
-                parm: [{ fileName: "DS_DIR", isDirty: true, uri: vscode.Uri.parse("") }],
-                mock: [
-                    {
-                        spy: jest.spyOn(ZoweSaveQueue, "push"),
-                        arg: [
-                            {
-                                fileProvider: "ds",
-                                savedFile: { fileName: "DS_DIR", uri: vscode.Uri.parse(""), isDirty: true },
-                                uploadRequest: saveFile,
-                            },
-                        ],
-                    },
-                ],
-            },
-            {
-                name: "onDidSaveTextDocument:3",
-                parm: [{ fileName: "USS_DIR", isDirty: true, uri: vscode.Uri.parse("") }],
-                mock: [
-                    {
-                        spy: jest.spyOn(ZoweSaveQueue, "push"),
-                        arg: [
-                            {
-                                fileProvider: "uss",
-                                savedFile: { fileName: "USS_DIR", isDirty: true, uri: vscode.Uri.parse("") },
-                                uploadRequest: saveUSSFile,
-                            },
-                        ],
-                    },
-                ],
-            },
             {
                 name: "onDidSaveTextDocument:4",
```

Comment:
I know this testing scheme was far from ideal even though it helped simplify things at the time.
However, now that we have removed a few tests (`it`s) from the list could we update this to `onDidSaveTextDocument:2`

Sorry for being picky 😢

Reply:
No worries, sorry I should have caught this while modifying these tests - addressed in cc4c0c40